mjcf_to_sdformat/tests/test_add_mjcf_geometry_to_sdf.py

```diff
@@ -0,0 +1,197 @@
+# Copyright (C) 2022 Open Source Robotics Foundation
+
+# Licensed under the Apache License, version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from numpy.testing import assert_allclose
+
+import sdformat as sdf
+from ignition.math import Vector3d
+from dm_control import mjcf
+
+from mjcf_to_sdformat.converters import geometry as geometry_conv
+
+import sdformat_mjcf_utils.sdf_utils as su
+
+GeometryType = sdf.Geometry.GeometryType
+
+
+class GeometryTest(unittest.TestCase):
+
+    def test_box(self):
+        x_size, y_size, z_size = 1, 2, 3
+
+        mujoco = mjcf.RootElement(model="test")
+        body = mujoco.worldbody.add('body')
+        geom = body.add('geom', type="box", name="box",
+                        size=[x_size, y_size, z_size])
+        sdf_geom = geometry_conv.add_mjcf_geometry_to_sdf(geom)
+
+        self.assertEqual(GeometryType.BOX, sdf_geom.type())
+        self.assertNotEqual(None, sdf_geom.box_shape())
+        self.assertEqual(Vector3d(x_size, y_size, z_size) * 2,
+                         sdf_geom.box_shape().size())
+
+    def test_capsule(self):
+        radius = 5.
+        length = 20.
+
+        mujoco = mjcf.RootElement(model="test")
+        body = mujoco.worldbody.add('body')
+        geom = body.add('geom', type="capsule", size=[radius, length])
+        sdf_geom = geometry_conv.add_mjcf_geometry_to_sdf(geom)
+
+        self.assertEqual(GeometryType.CAPSULE, sdf_geom.type())
+        self.assertNotEqual(None, sdf_geom.capsule_shape())
+        self.assertEqual(radius, sdf_geom.capsule_shape().radius())
+        self.assertEqual(length, sdf_geom.capsule_shape().length() / 2)
+
+    def test_capsule_fromto(self):
+        radius = 5.
+        length = 1.
+
+        mujoco = mjcf.RootElement(model="test")
+        body = mujoco.worldbody.add('body')
+        geom = body.add('geom', type="capsule", size=[radius],
```

Comment:
Can you add a test for `fromto` where it affects the pose?

Reply:
https://github.com/gazebosim/gz-mujoco/pull/26/commits/21b76c3da90666223352811af70d216f8f722e68